data/presets/barrier/wicket_gate.json

```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-gate",
+    "geometry": [
+        "vertex",
+        "line"
+    ],
+    "fields": [
+        "access",
+        "locked",
+        "wheelchair",
+        "opening_hours",
+        "height",
+        "material"
+    ],
+    "moreFields": [
+        "colour",
+        "manufacturer",
+        "operator",
+        "ref"
```

Comment:
All of these fields (except `access`), currently only have zero or one usage worldwide (out of 2k features matching the preset). :thinking:

Reply:
I just copy-pasted the fields from the Gate preset since I thought that the same fields could apply to wicket gates. I've narrowed down the fields to just the ones listed [on the wiki](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dwicket_gate) (`access`, `entrance`, and `opening_hours` (however it looks like `opening_hours` currently [only has 5 uses](https://overpass-turbo.eu/s/1jvZ).

Reply:
I'd say if there are only so few features with this combination, it's probably better to drop the field for now.